discovery.md

```diff
@@ -650,30 +635,49 @@ upon return of the `202 Accepted` from the original request.
 
 #### `POST /services`
 
-This MUST add the specified Services to the list of Services available at this
-Discovery Endpoint. The Body of the request message MUST contain an array of
-zero or more Service Entries. For convenience to clients, each Service in the
-array MAY include the `id` or `epoch` attributes but they MUST be ignored by
-the Discovery Endpoint. Both of these attributes' values MUST be defined by
-the Discovery Endpoint.
+This MUST update the collection of Services in the Discovery Endpoint to
+contain all Services in the request or fail.  Any existing Service identified
+in the request will be completely replaced by the Service definition in the
+request.  The body of the request message MUST contain an array of zero or more
+Service entries.
+
+The `id` and `epoch` attributes of the Services MAY be omited. In such cases
```

Comment:
s/omited/omitted/ I think

Reply:
that is correct, thank you for catching the spelling error